src/project/ProjectNameEditable.tsx

```diff
@@ -36,16 +37,16 @@ const ProjectNameEditable = () => {
     }
   }, [dialogs, actions, project, intl]);
   return (
-    <HStack spacing={2.5}>
+    <HStack spacing={2.5} height="52px">
```

Comment:
Matt to play and see if we can avoid.

Reply:
@microbit-robert I've kept the pixel height but moved it up to the outside so it can be the same and shared with the other header (the 62px one).

Reply:
Oops, 68px.